courses/models.py

```diff
@@ -81,8 +81,12 @@ def get_course_language(course_id):
 
     @property
     def first_university(self):
```

Comment:
Since this makes an SQL query, I suggest we do not use a property; a property implies that very little calculation is made to compute the result. I suggest instead:

    def get_first_university(self):